assets/getting-started-content/activate.cljs

```diff
@@ -0,0 +1,48 @@
+(ns activate
+  (:require ["vscode" :as vscode]
+            [promesa.core :as p]
+            [joyride.core :as joyride]))
+
+;; This is the Joyride User `activate.cljs` script. It will run
+;; as the first thing when Joyride is activated, making it a good
+;; place for you to initialize things. E.g. install event handlers,
+;; print motivational messages, or whatever.
+
+;; You can run this and other User scripts with the command:
+;;   *Joyride: Run User Script*
+
+(comment
+  ;; Use Calva to start the Joyride REPL and connect it. Then
+  ;; evaluate some code in this Rich comment block. Or just write
+  ;; sme new code and evaluate that. Or whatever. It's fun!
+
+  ;; New to Calva and/or Clojure? Use the Calva command:
+  ;;   *Fire up the Getting Started REPL*
+  ;; It will guide you through the basics.
+
+  (-> 4
+      (* 10)
+      (+ 1)
+      inc)
+
+  (p/-> (vscode/window.showInformationMessage "Be a Joyrider 🎸" "Yes" "Of course!")
```

Comment:
Ah, now I get it you want to use the return value from showInfoMsg.

Why not use p/let?

```
(p/let [choice (vscode/... ...)]
  (if choice ...))
```